### ISSUE TITLE
Some URLs not working when deployed in sub-location

### ISSUE THREAD

**Issue by harbulot** [verdaccio/verdaccio#384]
#### My reason:

When Verdaccio is deployed behind a reverse-proxy in a sub-location (`/npm` or `/verdaccio`, as described in the reverse proxy documentation), the `src` URL for the logo in the `<img />` tag in the webpage header ignores the `url_prefix` and returns a path starting at `/`.

The addresses in the header (`npm set registry https://...`) also ignore the sub-location.

#### Steps to reproduce:

Apache Httpd front-end, configured as a reverse proxy:

```
<VirtualHost *:80>
  AllowEncodedSlashes NoDecode
  ProxyPass /verdaccio http://127.0.0.1:4873 nocanon
  ProxyPassReverse /verdaccio http://127.0.0.1:4873
</VirtualHost>
```

Go to `https://my.host.example/verdaccio/` with a browser.

####  App Version:	

2.6.1

####  Config file:	

```
url_prefix: 'https://my.host.example/verdaccio/'
```

or 

```
url_prefix: '/verdaccio/'
```

#### Additional information:

See: https://github.com/verdaccio/verdaccio/blob/master/src/webui/src/components/Header/index.js#L144

```
<Link to="/">
    <img src={ this.state.logo } className={ classes.logo } />
</Link>
```

Here, it looks like `API.get('logo')` (used above in the same file) doesn't take `url_prefix` or the sub-location into account.

The same problem happens when using `origin` just below:

```
npm set registry { location.origin }
<br/>
npm adduser --registry { location.origin }
```

That said, the repository, if set with `repository=https://my.host.example/verdaccio/` in `.npmrc` works, at least as a cache for other repositories. This is certainly not a major problem if this is limited to the index page presentation, but it could be a more general problem if it happens in other places in the code.

**Comment by juanpicado** [comment]
Thanks for reporting. I guess @Meeeeow might have an idea about this.

**Comment by Meeeeow** [comment]
Sorry for the delay, i did not see this one in my timeline, i will look into it tomorrow

**Comment by Meeeeow** [comment]
Fixed in #393 

/cc @juanpicado

**Comment by juanpicado** [comment]
Shipped `v2.6.4`
